examples/map_transformations/reprojection_carrington.py

```diff
@@ -24,32 +22,23 @@
 
 plt.figure()
 aia_map.plot()
-plt.show()
```

Comment:
Do we not need a show at each point?

Reply:
I don't want to block this PR on that, so we can ignore my comment and punt it to a weekly meeting or the doc sprint.

Reply:
Created #6561 for continued discussion